barretenberg/cpp/src/barretenberg/avm_fuzzer/tx_fuzzer_lib.cpp

```diff
@@ -81,10 +81,11 @@ SimulatorResult fuzz_tx(FuzzerTxData& tx_data)
 }
 
 // Initialize FuzzerTxData with sensible defaults
-FuzzerTxData create_default_tx_data()
+FuzzerTxData create_default_tx_data(std::mt19937_64& rng)
 {
+    FuzzerData fuzzer_data = generate_fuzzer_data(rng);
```

Comment:
do we want rng at this point? Won't this create different `default_tx_data`. usually this is only called on the first iteration